src/generators/utils/parse.ts

```diff
@@ -47,3 +55,117 @@ export function parseSitesFixesConfig<T extends SiteProps>(text: string, options
 
     return sites;
 }
+
+// URL patterns are guaranteed to not have protocol and leading '/'
+function getDomain(url: string) {
+    return url.split('/')[0].toLowerCase();
+}
+
+function encodeOffsets(offsets: number[]): string {
```

Comment:
I think we should have at this place a good big comment explaining this magic about storing the offsets into a string.

Reply:
Sure thing. I changed the method a bit and added a comment explaining how it works.